autoscheduler/frontend/src/types.ts

```diff
@@ -1,3 +1,48 @@
+/* eslint-disable import/prefer-default-export */
+
 /**
  * This file is where we will declare the types of each data-holding class we will be using.
  */
+export enum MeetingType {
+  LEC, LAB, RES, INS
```

Comment:
Should include all of the possible meeting types, see: [Meeting Types](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/wiki/Meeting-Types)